src/plugin/withAndroidSourcemaps/withAndroidSourcemaps.ts

```diff
@@ -14,11 +14,13 @@ const withAndroidSourcemaps: ConfigPlugin<void> = (config) => {
       return config;
     }
 
+    const sourcemapsGradlePath = `${require("path").dirname(
+      require.resolve("@datadog/mobile-react-native/package.json")
+    )}/datadog-sourcemaps.gradle`.replace(/\\/g, "/");
```

Comment:
normally, file path separator is OS-specific, so we can use `File.separator` instead of hardcoding it. But not sure how much it is an issue.

Reply:
ah, this is TypeScript and I referenced JDK API thinking it is a Groovy script 🤦

Reply:
All good, I have still learnt something new, `File.separator` could come in handy in the future 😄